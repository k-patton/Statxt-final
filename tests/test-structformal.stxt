struct book {
	string title;
	string author;
	
};

struct book f(struct book b){
	printstr(b.title);

	b.title = "Sorcerer's Stone";

	return b;
}

int main() {
	struct book b1;
	b1.title = "Harry Potter";
	b1.author = "JK Rowling";

	int i;
	i = 1;
	b1 = f(b1);


	printstr(b1.title);

	i = strlen(b1.title);

	print(i);

	return 0;
	
}