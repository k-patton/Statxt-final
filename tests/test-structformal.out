Harry Potter
Sorcerer's Stone
16
